common/workunit/workunit.cpp

```diff
@@ -2695,6 +2695,39 @@ cost_type aggregateDiskAccessCost(const IConstWorkUnit * wu, const char *scope)
     }
     return totalCost;
 }
+
+void gatherSpillSize(const IConstWorkUnit * wu, const char *scope, stat_type & totalSizeSpill, stat_type & peakSizeSpill)
```

Comment:
Should probably have a comment that total and peak should be initialised before calling, since that is slightly unusual.